src/Service.Tests/Unittests/ConfigFileWatcherUnitTests.cs

```diff
@@ -159,5 +198,164 @@ public void HotReloadConfigRestRuntimeOptions()
                 File.Delete(configName);
             }
         }
+
+        /// <summary>
+        /// Creates a hot reload scenario in which the schema is wrong which causes
+        /// hot reload to fail, then we check that the program is still able to work
+        /// properly by showing us that it is still using the same configuration file
+        /// from before the hot reload.
+        /// </summary>
+        [TestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        public void HotReloadValidationFail()
```

Comment:
These two new tests don't really belong in ConfigFileWatcherUnitTests since they aren't related to ConfigFileWatcher